Project/feature-u-soundboard-api/src/controllers/soundController.ts

```diff
@@ -5,20 +5,20 @@ import fs from 'fs';
 import {randomUUID} from "crypto";
 
 class SoundController {
-    get(username:string) {
-        return getSounds(username);
+    async get(username:string) {
+        return await getSounds(username);
```

Comment:
is dit nu niet dubbelop als je hierbinnen en buiten in de index await doet

Reply:
Ik weet niet zeker of het perse nodig is om hier ook async await te gebruiken. Het kan echter sws geen kwaad (miss allen performance? maar dat zou een compiler kunnen solve als dat zo is).

Het geeft sws wat expliciter aan dat dit stuk asynchroon is wat wel chil is naar mijn mening.